### ISSUE TITLE
Not working?

### ISSUE THREAD

**Issue by nerdtron123** [zwopple/PocketSocket#68]
I've created an instance of PSWebSocketServer, set the delegate, and started, using example methods that NSLog. Now, when I try to open a websocket via javascript, it times out after 10 seconds.

**Comment by kisileno** [comment]
Server is working fine for me.

**Comment by nerdtron123** [comment]
On iOS?

**Comment by kisileno** [comment]
No, I am talking about macOS app with a socket server inside app and regular js client in a WKWebView.

**Comment by nerdtron123** [comment]
Does it not work on iOS?
